.travis.yml

```diff
@@ -38,19 +31,21 @@ branches:
 before_install:
   - pip install -U pip setuptools wheel
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - openwisp-utils-qa-checks --migrations-to-ignore 4 --migration-path ./django_x509/migrations/
   - npm install -g jslint
 
 # command to install requirements
 install:
   - pip install $DJANGO
   - python setup.py -q develop
 
+# qa checks
+before_script:
+  - openwisp-utils-qa-checks --migrations-to-ignore 4 --migration-path ./django_x509/migrations/ --migration-module django_x509
```

Comment:
I moved this here because pending migrations check was failing as django was not installed in before_install

Reply:
it's good, thanks